src/renderers/shared/ReactDebugTool.js

```diff
@@ -75,7 +133,9 @@ function getTreeSnapshot(registeredIDs) {
       updateCount: ReactComponentTreeHook.getUpdateCount(id),
       childIDs: ReactComponentTreeHook.getChildIDs(id),
       // Text nodes don't have owners but this is close enough.
-      ownerID: ownerID || ReactComponentTreeHook.getOwnerID(parentID),
+      ownerID: ownerID ||
+        parentID && ReactComponentTreeHook.getOwnerID(parentID) ||
```

Comment:
I tried to add an invariant saying that if ownerID is null, enforce that parentID is not null, but it throws in tons of places in tests.